tests/models/test_dag.py

```diff
@@ -2652,16 +2637,19 @@ def test_resolve_documentation_template_file_rendered(self):
             """
             )
             f.flush()
+            template_dir = os.path.dirname(f.name)
             template_file = os.path.basename(f.name)
 
-            with DAG("test-dag", start_date=DEFAULT_DATE, doc_md=template_file) as dag:
-                task = EmptyOperator(task_id="op1")
-
-                task
+            @dag_decorator(
+                "test-dag", start_date=DEFAULT_DATE, template_searchpath=template_dir, doc_md=template_file
```

Comment:
This test was broken even after correcting the assertion. Just needed a proper `template_searchpath` arg.